homeassistant/components/purpleair/config_flow.py

```diff
@@ -59,6 +67,20 @@ def async_get_coordinates_schema(hass: HomeAssistant) -> vol.Schema:
     )
 
 
+@callback
+def async_get_nearby_sensors_options(
+    nearby_sensor_results: list[NearbySensorResult],
+) -> list[SelectOptionDict]:
+    """Return a set of nearby sensors as SelectOptionDict objects."""
+    return [
+        SelectOptionDict(
+            value=str(result.sensor.sensor_index),
+            label=f"{result.sensor.name} ({round(result.distance, 1)} km away)",
```

Comment:
I'd avoid including language besides names in the options as they can't be translated.

Reply:
https://github.com/home-assistant/core/pull/84893